app/models/company.rb

```diff
@@ -48,4 +48,8 @@ def latest_assessment
   def to_s
     name
   end
+
+  def isin_string
```

Comment:
I think this belongs to `CompanyDecorator` which needs to be created :)

Reply:
also, this name is a bit confusing - this returns an array.

Reply:
you're right! I've moved the method to a new decorator, and I've renamed it to `isin_as_tags`. 🙇

Reply:
👍